File: ordering_backend/src/main/java/com/lanut/ordering_backend/entity/Order.java
package com.lanut.ordering_backend.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDateTime;

/**
 * <p>
 * 
 * </p>
 *
 * @author lanut
 * @since 2024-06-04
 */
@Data
public class Order implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "order_id", type = IdType.AUTO)
    private Integer orderId;

    private Integer userId;

    private LocalDateTime orderDate;

    private BigDecimal totalAmount;

    private String orderStatus;

    @Override
    public String toString() {
        return "Order{" +
        "orderId = " + orderId +
        ", userId = " + userId +
        ", orderDate = " + orderDate +
        ", totalAmount = " + totalAmount +
        ", orderStatus = " + orderStatus +
        "}";
    }
}

File: ordering_backend/src/main/java/com/lanut/ordering_backend/entity/Order.kt
package com.lanut.ordering_backend.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import java.io.Serializable
import java.math.BigDecimal
import java.time.LocalDateTime

/**
 * <p>
 * 
 * </p>
 *
 * @author lanut
 * @since 2024-06-04
 */
class Order : Serializable {

    @TableId(value = "order_id", type = IdType.AUTO)
    var orderId: Int? = null

    var userId: Int? = null

    var orderDate: LocalDateTime? = null

    var totalAmount: BigDecimal? = null

    var orderStatus: String? = null

    override fun toString(): String {
        return "Order{" +
        "orderId=" + orderId +
        ", userId=" + userId +
        ", orderDate=" + orderDate +
        ", totalAmount=" + totalAmount +
        ", orderStatus=" + orderStatus +
        "}"
    }
}
